tests/documents.test.ts

```diff
@@ -144,7 +144,10 @@ describe('Documents tests', () => {
 
       test(`${permission} key: Get documents with filters`, async () => {
         const client = await getClient(permission);
-        await client.index(indexPk.uid).updateFilterableAttributes(['id']);
+        client.waitForTask(
+          (await client.index(indexPk.uid).updateFilterableAttributes(['id']))
+            .taskUid,
+        );
```

Comment:
Out of curiosity, why did you make this change?

And if it's really needed, can you split this into 2 lines for readability? 🙏

Reply:
Because `updateFilterableAttributes` return an `EnqueuedTask` that needs to be "awaited" via `waitForTask`. Without it it's not entirely guaranteed that things will run in the order we want them to, even though so far they did.

Reply:
Oh, but I didn't await `waitForTask` 😅.